shell/platform/tizen/flutter_tizen_shell.cc

```diff
@@ -0,0 +1,121 @@
+#include <assert.h>
+#include <unistd.h>
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
+#include "flutter/shell/platform/tizen/public/flutter_tizen.h"
+
+extern int gApp_width;
+extern int gApp_height;
+
+std::string TPK_ROOT_PATH = "/tpkroot";
+std::string LIB_PATH = "/lib";
+std::string RES_PATH = "/res";
```

Comment:
Against the naming rules. Why don't you make this path thing consistent with `flutter_tizen_engine_test.cc`?

Reply:
It's mistake~ Sorry!